editor/editor_file_system.h

```diff
@@ -236,7 +245,8 @@ class EditorFileSystem : public Node {
 	List<String> sources_changed;
 	List<ItemAction> scan_actions;
 
-	bool _update_scan_actions();
+	void _update_scan_actions();
+	bool _update_scan_actions_helper();
```

Comment:
Feels like these two method names should be swapped tbh, but that would add more changes, so it's not important.

Reply:
Yeah, a lot of the names in `editor_file_system` aren't too descriptive IMO. I have a previous commit that documented tons of code, but I removed most of the comments because @willnationsdev mentioned they may end up becoming junk comments if they aren't properly maintained alongside changes.